integration/mediation-tests/tests-transport/src/test/java/org/wso2/carbon/esb/jms/inbound/transport/test/JMSInboundMessagePollingTestCase.java

```diff
@@ -45,6 +51,22 @@
 	private InboundAdminClient inboundAdminClient;
 	private ActiveMQServer activeMQServer = new ActiveMQServer();
 
+	String message = "<?xml version='1.0' encoding='UTF-8'?>" +
```

Comment:
Let's add access modifiers
Do we really need to make this class variable? I see only used in one function? or you planned for ahead :)

Reply:
Just added for the clarity. Di we need to care of access modifiers? This is a test class.

Reply:
Better adding for perfection, It doesn't matter test, dummy, sample, product, code :D